modules/Kconfig.mcux

```diff
@@ -25,6 +25,11 @@ config HAS_MCUX_LPADC
 	help
 	  Set if the LPADC module is present in the SoC.
 
+config HAS_LPC_ADC
+	bool
+	help
+	  Set if the LPC ADC module is present in the SoC.
+
```

Comment:
This change should be moved to the commit `drivers: adc: added support adc drive for lpc51u68` or a new commit